### PR TITLE
added PUT POST DEL operations, user seeds and PUT test.

### PR DESCRIPTION
**Pull request by Cberumen51**
In this PR the rest of the CRUD operations were finished, added some additional seeds for users and some tests.

- added the PUT, POST, and DELETE CRUD operations for the `/locations` route

- added users seeds which consists of 2 clients and 3 groomers 

- added test to the PUT operation for /locations to test that changes are actually being done

- added address, city, state and zip code to the migrations table for locations and profiles

### CODE REVIEW

File: __tests__/routes/locations.test.js [hunk 1]
```diff
@@ -63,4 +63,41 @@ describe('locations router endpoints', () => {
...
+        .put(`/locations/hgf5rfgh54`)
+        .send(location);
+      const first = res.body.data[0];
+      console.log([res.status, res]);
+
+      expect(res.status).toBe(200);
+      expect(first.groomerId).not.toBe(`hgf5rfgh54`);
+      expect(first.groomerId).toBe(`245feafeffehes`);
+    });
+  });
```
Comment (dvwhite) [inline __tests__/routes/locations.test.js:102]: Well done implementing tests for your route handler.

File: __tests__/routes/locations.test.js [hunk 2]
```diff
@@ -63,4 +63,41 @@ describe('locations router endpoints', () => {
       expect(res.body.message).toBe('Location Not Found');
     });
   });
+
+  describe('PUT /locations', () => {
```
Comment (hmerritt) [inline __tests__/routes/locations.test.js:67]: Testing is never a bad thing!

**Review by hmerritt — APPROVED** [review]
Nice work adding useful CRUD operations plus tests to go with it!

**Review by dvwhite — COMMENTED** [review]
Great job building the model function, route handlers, tests, seeds, and migrations!

**Review by dvwhite — APPROVED** [review]

**Pull request merged**